### DISCUSSION TITLE
Turning off key/value padding in vertical hash

### DISCUSSION THREAD

**Discussion by msandoval** [Camelcade/Perl5-IDEA#2367] [Q&A]
Hi all, 

While I love nicely padded key/values in vertically written hashes, apparently the place I work at has an issue with them. Is there a way to turn this functionality off so that it doesn't auto prettify them?

Thank you,
Manuel

**Comment by hurricup** [comment]
If I get this right, then yes, you can

![image](https://user-images.githubusercontent.com/2811330/116504682-8fd22300-a8c1-11eb-9d0f-2fd1967cdc17.png)

**Reply by msandoval** [reply]
Thank you! This worked. I appreciate your help!
